pymongo/client_options.py

```diff
@@ -144,7 +144,11 @@ def _parse_ssl_options(options: Mapping[str, Any]) -> tuple[Optional[SSLContext]
 
 
 def _parse_pool_options(
-    username: str, password: str, database: Optional[str], options: Mapping[str, Any]
+    username: str,
+    password: str,
+    database: Optional[str],
+    options: Mapping[str, Any],
+    is_async: bool,
```

Comment:
Given we use `_IS_SYNC` everywhere, can we rename this to "is_sync" for consistency?

Reply:
Done!